src/components/Recommendations/index.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import useRecommendations from "../../hooks/useRecommendations";
+import MovieCard from "../MovieCard";
+import {
+    RecommendationsMain,
+    RecommendationsContainer as Container,
+    Title,
+} from "./styles";
+
+export default function Recommendations({ id, type }) {
+    const { data, isLoading, isError } = useRecommendations(id, type);
+
+    if (isLoading) return "Loading Recommendations...";
+
+    if (isError) return "Error Recommendations";
+
+    return (
+        <RecommendationsMain>
+            <Title>
+                {data.results.length !== 0
+                    ? "More Like This"
+                    : "No Movie Recommendations"}
+            </Title>
+            {data.results.length !== 0 && (
```

Comment:
How bout this instead?
```javascript
{data.results.length && (
```
if the length results to 1 or above then it is a truthy value. When it is 0, it is a falsy value.